beetsplug/lyrics.py

```diff
@@ -605,6 +638,8 @@ def _scrape_strip_cruft(html, plain_text_out=False):
     html = BREAK_RE.sub("\n", html)  # <br> eats up surrounding '\n'.
     html = re.sub(r"(?s)<(script).*?</\1>", "", html)  # Strip script tags.
     html = re.sub("\u2005", " ", html)  # replace unicode with regular space
+    html = re.sub("<aside .+?</aside>", "", html)  # remove Google Ads tags
+    html = re.sub(r"</?(em|strong)[^>]*>", "", html)  # remove bold / italics
```

Comment:
I've been working on this and updated Tekstowo and Genius backends to make use of JSON data that can be found in the HTML response, something like this:

```python
class Genius(Backend):
    """Fetch lyrics from Genius via genius-api.

    Simply adapted from
    bigishdata.com/2016/09/27/getting-song-lyrics-from-geniuss-api-scraping/
    """

    REQUIRES_BS = True
    JSON_BLOCK_RE = re.compile(r"(?<=JSON.parse\(').*?(?='\);\n)", re.DOTALL)
    #: BACKSLASH_RE
    BACKSLASH_RE = re.compile(r"\\{1}(?!\\)|\\(?=\\\\)")

    base_url = "https://api.genius.com"
    search_url = f"{base_url}/search"

    def __init__(self, config, log):
        super().__init__(config, log)
        self.api_key = config["genius_api_key"].as_str()
        self.headers = {"Authorization": f"Bearer {self.api_key}"}

    def fetch(self, artist, title, album=None, length=None):
        """Fetch lyrics from genius.com

        Because genius doesn't allow accessing lyrics via the api,
        we first query the api for a url matching our artist & title,
        then attempt to scrape that url for the lyrics.
        """
        if (url := self.find_lyrics_url(artist, title)) and (
            m := self.JSON_BLOCK_RE.search(self.fetch_text(url))
        ):
            data = json.loads(self.BACKSLASH_RE.sub("", m.group()))
            html = data["songPage"]["lyricsData"]["body"]["html"]
            return get_soup(html).get_text()

        return None
```

This will most probably render those html regexes redundant. 

But I think I will leave this out for another PR since this one is already getting out of hand regarding the size 😅

Reply:
Alright, that's great news!  Yeah, we can leave it for another PR.